Trine.Analyzer.Tests/EnumValueTests.cs

```diff
@@ -0,0 +1,64 @@
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
```

Comment:
Use NUnit instead for consistency?

Reply:
This project is generated by VS, so it's using Microsoft UnitTesting allover the place. Could try to migrate to NUnit but probably not in this PR though.